recipes-bsp/atf/atf-sunxi_git.bb

```diff
@@ -1,18 +1,24 @@
 DESCRIPTION = "ARM Trusted Firmware Allwinner"
 LICENSE = "BSD-3-Clause"
 LIC_FILES_CHKSUM = "file://license.md;md5=829bdeb34c1d9044f393d5a16c068371"
+LIC_FILES_CHKSUM:sun50i-h616 = "file://docs/license.rst;md5=b2c740efedc159745b9b31f88ff03dde"
 
-SRC_URI = "git://github.com/apritzel/arm-trusted-firmware;nobranch=1;protocol=https"
+SRC_URI = " \
+git://github.com/apritzel/arm-trusted-firmware;nobranch=1;protocol=https \
+file://0001-Use-same-type-as-in-declaration.patch \
+"
 SRCREV = "aa75c8da415158a94b82a430b2b40000778e851f"
 
-SRC_URI:append = " file://0001-Use-same-type-as-in-declaration.patch"
+SRC_URI:sun50i-h616 = "git://github.com/ARM-software/arm-trusted-firmware;nobranch=1;protocol=https"
```

Comment:
Would not make distinguish for h616 and other. I plan to move atf sunxi to mainline so it can be done now.

Reply:
So if I remove "apritzel" fork and keep only the mainline, will it work out-of-the-box for other sun50i-based boards or do we have to change something more?

Reply:
If you update it I'll try on  other board I have in stock. I think I've tried it once and it works so I didn't expect issues.